test/jdk/sun/security/krb5/auto/ReplayCacheTestProc.java

```diff
@@ -117,6 +119,15 @@ public static void main0(String[] args) throws Exception {
             Ex[] result;
             int numPerType = 2; // number of acceptors per type
 
+            // User-provided libs
+            String userLibs = System.getProperty("test.libs");
+            Asserts.assertNotNull(userLibs, "test.libs property must be provided");
+            libs = userLibs.split(",");
+            if (Arrays.asList(libs).contains("N") && !isNativeLibAvailable()) {
+                System.out.println("Native mode not available - skipped");
```

Comment:
Can you please add a comment here with which scenarios are skipped?

Reply:
Added an extra comment there.